webpack.config.js

```diff
@@ -25,7 +25,6 @@ module.exports = (env = {}) => { // pass command line arguments like `webpack ..
             __dirname: false,
             __filename: false,
             fs: false,
-            process: false,
```

Comment:
Wait we need this line here, or to figure out a way to make it work for both of us.